### ISSUE TITLE
Display Meta-Information of a pipeline

### ISSUE THREAD

**Issue by sroidl** [sroidl/lambda-ui#83]
Displaying meta-information of a pipeline would be a good feature to allow having more information for the pipeline.

One example could be step descriptions, that allow the pipeline creator to give detailed information on what a build step is supposed to do.

**Comment by hamvocke** [comment]
Will this also include displaying information about the git commit that triggered the build (e.g. author, commit message and SHA) or should this be another issue?

**Comment by flosell** [comment]
FYI, since 0.13.0, LambdaCD supports build-level metadata that could be used for exactly this purpose. See the [wiki](https://github.com/flosell/lambdacd/wiki/Build%20Metadata) and flosell/lambdacd#138. Metadata on a per-step level is planned but not yet implemented (flosell/lambdacd#86).

(by the way, lambda-ui doesn't support 0.13.0 yet, see PR #90 to fix this)

**Comment by hamvocke** [comment]
Sweet! So we'd have to fetch git commit data and store it as build-metadata. I assume [lambdacd-git](https://github.com/flosell/lambdacd-git) is the right place for this functionality?

**Comment by flosell** [comment]
Exactly, lambdacd-git would probably be the right place to put this. It should already have decent support for getting git-commit data as it is and there's an open issue to add support for metadata: flosell/lambdacd-git#20

I just added some thoughts on different ways to approach this, feel free to give feedback
